Doc/library/unittest.rst

```diff
@@ -1446,17 +1449,19 @@ along with their deprecated aliases:
     :meth:`.assertAlmostEqual`      failUnlessAlmostEqual  assertAlmostEquals
     :meth:`.assertNotAlmostEqual`   failIfAlmostEqual      assertNotAlmostEquals
     :meth:`.assertRegex`                                   assertRegexpMatches
+    :meth:`.assertNotRegex`                                assertNotRegexpMatches
```

Comment:
Yes, I changed that, after Brett's suggestion, to state it was introduced and deprecated at 3.5. It has to have an entry in this table so people who are porting testing code from 2 to 3 can see the new name for it.